posframe.el

```diff
@@ -676,20 +677,15 @@ WIDTH and MIN-WIDTH."
   (posframe--kill-buffer buf-or-name))
 
 (defun posframe-delete-frame (buf-or-name)
-  "Kill child-frame of posframe.
-This posframe's buffer is BUF-OR-NAME."
-  (dolist (frame (frame-list))
-    (let ((buffer-info (frame-parameter frame 'posframe-buffer))
-          (buffer (get-buffer buf-or-name)))
-      (when (or (equal buf-or-name (car buffer-info))
-                (equal buf-or-name (cdr buffer-info)))
-        (when buffer
-          (with-current-buffer buffer
-            (dolist (timer '(posframe--refresh-timer
-                             posframe--timeout-timer))
-              (when (timerp timer)
-                (cancel-timer timer)))))
-        (delete-frame frame)))))
+  "Delete posframe pertaining to BUF-OR-NAME."
```

Comment:
when posframe buffer is killed, this commend will not work

Reply:
Sorry, I can't figure out what you mean by this: when the buffer is killed, this will just return nil, which seems the right thing?